src/common/span.rs

```diff
@@ -139,7 +150,13 @@ impl Debug for Span {
     // TODO: use the field, etc. constructor.
     fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
         if !self.is_empty() {
-            write!(f, "Span {{ {:?}, ({}, {}) }}", self.contents(), self.offset, self.length)
```

Comment:
Same here: multiple lines doesn't add much. I know this was probably automatically formatted, but this is why I like to configure my automatic formatters.